Doc/library/multiprocessing.rst

```diff
@@ -803,10 +803,13 @@ Miscellaneous
 Connection Objects
 ~~~~~~~~~~~~~~~~~~
 
+.. currentmodule:: None
+
 Connection objects allow the sending and receiving of picklable objects or
 strings.  They can be thought of as message oriented connected sockets.
 
-Connection objects are usually created using :func:`Pipe` -- see also
+Connection objects are usually created using
+:func:`Pipe <multiprocessing.Pipe>` -- see also
 :ref:`multiprocessing-listeners-clients`.
 
 .. class:: Connection
```

Comment:
Maybe add the `:noindex:` option?

Reply:
Oh, I wanted just exclude it from the index, but seems this doesn't work this way.